AutomationTests/Malafi.Tests/Steps/DocumentTypeFeatureStepDefinitions.cs

```diff
@@ -70,29 +72,136 @@ public void WhenIClickOnViewApprovalLevelsButton()
-            Thread.Sleep(100);  
+            var IDApprovallevelList = scenarioContext["IDApprovallevelList"] as IDApprovallevelList;
+            Thread.Sleep(100);
             Assert.IsNotNull(IDApprovallevelList);
             IDApprovallevelList.Wait.Until(ExpectedConditions.ElementToBeClickable(IDApprovallevelList.AddApprovalLevelClick));
-           
-            Assert.AreEqual("Add Approval level", IDApprovallevelList.AddApprovalLevelClick.Text);            
 
-           
+            Assert.AreEqual("Add Approval level", IDApprovallevelList.AddApprovalLevelClick.Text);
+
+
         }
 
         [When(@"I click on AddApprovalLevelClick button")]
         public void WhenIClickOnAddApprovalLevelClickButton()
         {
-            Thread.Sleep(100);  
+            Thread.Sleep(100);
             var NewDocTypeApprovalLVI = iDApprovallevelList.ClickOnAddApprovallevelLink();
             scenarioContext["NewDocTypeApprovalLVI"] = NewDocTypeApprovalLVI;
 
+        }
+
+        [Then(@"I should see Title in the top left corner")]
+        public void ThenIShouldSeeTitleInTheTopLeftCorner_()
+        {
+            Assert.AreEqual("Document Type List", documentTypesPage.CheckTitleDocumentsTypes.Text);
+
+        }
+
+        [Then(@"I should see search box and add document type button in the top right corner")]
+        public void ThenIShouldSeeSearchBoxAndAddDocumentTypeButtonInTheTopRightCorner_()
+        {
+            Assert.IsTrue(documentTypesPage.CheckSearchBox.Enabled);
+            Assert.IsTrue(documentTypesPage.CheckAddDocumentTypeButton.Enabled);
+            Assert.AreEqual("Add Document Type", documentTypesPage.CheckAddDocumentTypeButton.Text);
+
+        }
+
+        [Then(@"I should see for buttons Edit Delete Add it to Documents Group View approval levels")]
+        public void ThenIShouldSeeForButtonsEditDeleteAddItToDocumentsGroupViewApprovalLevels_()
+        {
+            Assert.IsTrue(documentTypesPage.CheckEditButton.Enabled);
+            Assert.IsTrue(documentTypesPage.CheckDeleteButton.Enabled);
+            Assert.IsTrue(documentTypesPage.CheckAddDocumentTypeButton.Enabled);
+            Assert.IsTrue(documentTypesPage.CheckViewButton.Enabled);
+
+
+        }
+
+        [Given(@"Click on sortable-icon")]
+        public void GivenClickOnSortable_Icon()
+        {
+            documentTypesPage.Wait.Until(ExpectedConditions.ElementToBeClickable(documentTypesPage.CheckSortaButton)).Click();  
+        }
+
+        [Then(@"I should see in table sortable-icon")]
+        public void ThenIShouldSeeInTableSortable_Icon()
+        {
+            Assert.IsTrue(documentTypesPage.CheckSortaButton.Enabled);
+
+        }
+
+        [Then(@"I should see table Title and Is Required and Has Expiry and Action and sortable-icon it works")]
+        public void ThenIShouldSeeTableTitleAndIsRequiredAndHasExpiryAndActionAndSortable_IconItWorks()
```

Comment:
This function contains too much lines of code, please try to divide it into some functions